pkg/image/prune/imagepruner_test.go

```diff
@@ -257,14 +257,14 @@ func bcList(bcs ...buildapi.BuildConfig) buildapi.BuildConfigList {
 	}
 }
 
-func bc(namespace, name string, strategyType buildapi.BuildStrategyType, fromKind, fromNamespace, fromName string) buildapi.BuildConfig {
+func bc(namespace, name, strategy string, fromKind, fromNamespace, fromName string) buildapi.BuildConfig {
```

Comment:
Please make the argument names consistent among `func bc()`, `func build()` and `func buildSpec()`. Use either `strategy` or `strategyType` in all of them.